sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/CapabilitiesScenarioTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.ResourceManager.Sql;
+using Azure.ResourceManager.Sql.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Sql.Tests
+{
+    public class CapabilitiesScenarioTests : SqlManagementClientBase
+    {
+        public CapabilitiesScenarioTests(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [Test]
+        public async Task TestGetCapabilitiesAsync()
+        {
+            Dictionary<string, string> tags = new Dictionary<string, string>();
+            string suiteName = this.GetType().Name;
+
+            SqlManagementClient sqlClient = GetSqlManagementClient();
+
+            LocationCapabilities capabilities = await sqlClient.Capabilities.ListByLocationAsync(DefaultLocation);
+
+            Assert.NotNull(capabilities);
+
+            foreach (ServerVersionCapability s in capabilities.SupportedServerVersions)
```

Comment:
should use descriptive variable names, for example

s => version
o => objective
m => range